app/services/idv/financials_step.rb

```diff
@@ -37,11 +37,5 @@ def vendor_params
       finance_type = idv_form.finance_type
       { finance_type => idv_form.idv_params[finance_type] }
     end
-
-    def extra_analytics_attributes
-      {
-        vendor: { reasons: vendor_reasons },
```

Comment:
Per the commit message: `Remove extra analytics attributes for vendor reasons in the Financials
and Phone steps because Equifax does not return any reasons for those
verifications.`